### PR TITLE
Render triangles correctly on the mod map

### PR DESCRIPTION
**Pull request by charliegroll**

### CODE REVIEW

File: public/js/modMap.js [hunk 1]
```diff
@@ -25,32 +24,26 @@ function modMap() {
...
-      t = checkIfModExists(allMods, x, i)
-      //console.log(t);
-      if (t) {
+
+      const mod = getMod(allMods, x, i)
+
+      if (mod) {
         var defaultColor = color
         var op = 1
-        var id = t["_id"]
```
Comment (charliegroll) [inline public/js/modMap.js:47]: @nkwsy is it `_id` or `id`? https://github.com/nkwsy/Wildmile/pull/37/files#diff-966cb6067f5126e5899afb7013f0b6adR118

File: views/modules.pug [hunk 2]
```diff
@@ -2,39 +2,13 @@ extends layout
 
 block content
   .page-header
-    h3 New Module
-
-  table#plants.table-striped.table-bordered.table-sm
```
Comment (charliegroll) [inline views/modules.pug:7]: Removing the whole table as per https://urbanrivers.slack.com/archives/C015RUE8ZQB/p1595367858015000

File: public/js/modMap.js (hunk 1 above)
Comment (nkwsy) [reply public/js/modMap.js:47]: it works both ways. but _id is more correct

File: public/js/modMap.js [hunk 3]
```diff
@@ -63,53 +56,28 @@ function modMap() {
...
-            y
-          }).stroke({
-            color: '#5ECCA2',
-            width: 1
-          })
 
-        }
-        if (t["shape"] == "R3") {
-          var rect = draw.rect(20, 60).attr({
+          coordinates = coordinates.join(' ');
```
Comment (nkwsy) [inline public/js/modMap.js:78]: I got an error here when a user has a `T3` with the orientation `flat` the page wouldn't render. Can you add something like this so it still renders but it shows the user is an idiot
```
          if (mod['orientation'] === 'flat') {
            coordinates = [topRight, bottomLeft];
            coordinates.push(flipped ? bottomRight : topLeft);
            let defaultColor = '#C73316';
```

Comment (charliegroll) [reply public/js/modMap.js:78]: what does `flat` mean?

Comment (nkwsy) [reply public/js/modMap.js:78]: Flat is for rectangles

Comment (charliegroll) [reply public/js/modMap.js:84]: dd2dd0a

File: public/js/modMap.js [hunk 4]
```diff
@@ -25,32 +24,26 @@ function modMap() {
   console.log(article.dataset);
   var allMods = JSON.parse(article.dataset.mods);
   var defaultColor = 'grey'
-  console.log(allMods[0]["x"]);
+  console.log('allMods: ', allMods);
 
   //var rect = draw.rect(60, 20).attr({ fill: '#189968' }).stroke({ color: '#5ECCA2',  width: 2 })
   var wide = 30;
   for (var n = 0; n < 30; n++) {
     for (var i = 0; i < 10; i++) {
```
[the remark this answers is not in the document]
Comment (charliegroll) [reply public/js/modMap.js]: 7518343

File: public/js/modMap.js [hunk 5]
```diff
@@ -25,32 +24,26 @@ function modMap() {
   console.log(article.dataset);
   var allMods = JSON.parse(article.dataset.mods);
   var defaultColor = 'grey'
-  console.log(allMods[0]["x"]);
+  console.log('allMods: ', allMods);
 
   //var rect = draw.rect(60, 20).attr({ fill: '#189968' }).stroke({ color: '#5ECCA2',  width: 2 })
   var wide = 30;
   for (var n = 0; n < 30; n++) {
```
[the remark this answers is not in the document]
Comment (charliegroll) [reply public/js/modMap.js]: 7518343

**Pull request merged**